rasa/nlu/tokenizers/convert_tokenizer.py

```diff
@@ -7,17 +7,27 @@
 from rasa.nlu.tokenizers.whitespace_tokenizer import WhitespaceTokenizer
 from rasa.shared.nlu.training_data.message import Message
 from rasa.utils import common
+from rasa.nlu import utils as nlu_utils
```

Comment:
minor: In our [code guidelines](https://www.notion.so/rasa/Python-Code-Conventions-26835ffd72484dd699f28fa693916d2e#4fbe14b7c3a04fbfa20fa0e76995ecf7) we suggested to use the absolute import instead of aliases

Reply:
Hmm, but for imports like those of util functions, shouldn't we use aliases to make the import clearer because we could be importing functions from multiple util modules like `rasa.nlu.utils`, `rasa.shared.utils`? If these functions are declared in the `__init__.py` of the respective modules(like in the case above) then imports will clash.

Reply:
Ahh I guess you mean importing it as `rasa.nlu.utils` and not `from rasa.nlu import utils` 👍

Reply:
There is a whole discussion about this somewhere in Slack and in Notion. In the end we decided to go for this. 

> If these functions are declared in the __init__.py of the respective modules(like in the case above) then imports will clash.

With which other things would that crash?

Reply:
> Ahh I guess you mean importing it as rasa.nlu.utils and not from rasa.nlu import utils 👍

Yes, exactly 👍